PSC-ONBOARDING-OFFBOARDING.md

```diff
@@ -0,0 +1,186 @@
+for further details on the PSC selection process.
+
+This selection process is initiated with an email sent to the
+[kubernetes-security-discuss mailing list](https://groups.google.com/forum/#!forum/kubernetes-security-discuss)
+once the three month Associate Role period is complete.
+
+The following template may be used:
+
+```
+I want to nominate Jane Doe to join the Product Security Committee as
+a full member. She has been an associate member since December 4th 2018
+and in that role over the last three months he has helped on a number of
+tasks including:
+
+<list work carried out as associate>
+
+Our process for accepting this nomination is lazy consensus and I will
+close the round of consensus on March 15th, 2019 to leave time for any
+comments or question. If the nomination is accepted Jane will take on
+the full responsibilities of a Security Committee member and take on a
+weekly rotation for security@kubernetes.io on-call rotation.
+```
+
+### Required file and access grant updates.
+
+Once the PSC has promoted an individual(s)  to a full PSC member, various
+systems require updating. Likewise, should a PSC member leave the PSC - the
+reverse is required, with the user being removed from each system.
+
+#### kubernetes/security repository
+
+All PSC member discussion & approval must happen on the `kubernetes/security`
+repository first and only once approved by means of the pull request being
+merged, should pull requests be approved / merged in the `kubernetes/community`
+repository.
+
+##### file: https://github.com/kubernetes/security/blob/master/security-release-process.md
+
+Add / remove the PSC member(s) github name from `community/sig-list.md`
+to the existing `committee-product-security` field, according to the usernames
+alphabetical placing.
+
+If adding a PSC member, you will also need to remove the associate role:
+
+```
+The initial Product Security Committee will consist of volunteers subscribed to the private [Kubernetes Security](https://groups.google.com/a/kubernetes.io/forum/#!forum/security) list. These are the people who have been involved in the initial discussion and volunteered:
+
+- Jane Doe (**[@philips](https://github.com/jdoe)**) `<jdoe@acme.com>` [GPG_KEY]
+```
+
+##### file: https://github.com/kubernetes/security/blob/master/OWNERS_ALIASES
+
+Add / remove the PSC member(s) github name from `security/OWNERS_ALIASES` to the
+existing `product-security-committee` field, according to the usernames
+alphabetical placing:
+
+```yaml
+aliases:
+  product-security-committee:
+    - jdoe
+```
+
+#### kubernetes/community repository
+
+##### file: https://github.com/kubernetes/community/blob/master/sigs.yaml
+
+Add / remove the PSC member(s)  github name from `community/sigs.yaml` to the
+existing `label`| `leadership` field, according to the usernames alphabetical
+placing:
+
+```yaml
+label: product-security
+  leadership:
+    chairs:
+    - github: jdoe
+      name: Jane Doe
+      company: Acme LTD
+```
```

Comment:
````suggestion
```

Once the yaml is updated, run `make generate` to regenerate the markdown versions.
````